test/api/v3/integration/hall/PUT-hall_heores_heroId.test.js

```diff
@@ -62,12 +62,14 @@ describe('PUT /heroes/:heroId', () => {
     expect(heroRes.contributor.level).to.equal(1);
     expect(heroRes.purchased.ads).to.equal(true);
     expect(heroRes.auth.blocked).to.equal(true);
+    expect(heroRes.preferences.sleep).to.equal(true);
```

Comment:
Pretty sure this data isn't passed back in the response, so this expectation shouldn't be here.